app/src/main/kotlin/com/ortin/gesturetranslator/app/di/CoreModule.kt

```diff
@@ -0,0 +1,45 @@
+package com.ortin.gesturetranslator.app.di
+
+import android.content.Context
+import com.ortin.gesturetranslator.core.managers.mediapipe.MediaPipeManager
+import com.ortin.gesturetranslator.core.managers.mediapipe.MediaPipeManagerImpl
+import com.ortin.gesturetranslator.core.managers.model_coordinate.ModelCoordinateManager
+import com.ortin.gesturetranslator.core.managers.model_coordinate.ModelCoordinateManagerImpl
+import com.ortin.gesturetranslator.core.repository.HandDetectionRepositoryImpl
+import com.ortin.gesturetranslator.core.repository.RecognizeCoordinateRepositoryImpl
+import com.ortin.gesturetranslator.domain.repository.HandDetectionRepository
+import com.ortin.gesturetranslator.domain.repository.RecognizeCoordinateRepository
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+class CoreModule {
+    @Provides
+    @Singleton
+    fun provideHandDetectionRepository(mediaPipeManager: MediaPipeManager?): HandDetectionRepository {
+        return HandDetectionRepositoryImpl(mediaPipeManager!!)
```

Comment:
```suggestion
    fun provideHandDetectionRepository(mediaPipeManager: MediaPipeManager): HandDetectionRepository {
        return HandDetectionRepositoryImpl(mediaPipeManager)
```

Ты же это уже исправлял... Раве нет?

Reply:
С твоими исправлениями...

Reply:
Сделано